docs/maintenance/Pull_Requests.mdx

```diff
@@ -111,6 +108,24 @@ For preliminary reviews, be clear with what scale you're reviewing at: _"Reviewi
 For repeat reviews, be clear when it's something you missed earlier and/or there's new information.
 Don't apologize if the missed information was only made clear because of requested changes - this is part of the review process!
 
+### Approvals
+
+If the PR addresses a time critical task, such as a security fix or `main` branch build break, go ahead and squash merge it.
+
+Otherwise, upon completing your review, if the build is passing and you have no blockers, approve it on GitHub.
+Then:
+
+- If there isn't a `1 approval` label or existing approval, add the `1 approval` label
+- If there's already `1 approval` and/or it's been a week since the last request for changes, go ahead and squash merge
+  - For straightforward PRs that don't impact users, you can wait 3 days instead
+
+We try to wait a week on PRs to give reviewers who aren't active every day a chance to get to it.
+
+:::tip
+If a PR has had `1 approval` and no change requests for a week, go ahead and merge it.
```

Comment:
This seems redundant? What's the purpose of this from the bullet above.

Reply:
My thought was that the instructions technically only apply if you are reviewing... but, yeah, I think anybody on the team should be able to understand this is implied. 👍